roster/views.py

```diff
@@ -409,6 +409,18 @@ def inquiry(request: AuthHttpRequest, student_pk: int) -> HttpResponse:
     return render(request, "roster/inquiry.html", context)
 
 
+@login_required
+@require_POST
+def cancel_inquiry(request: AuthHttpRequest, pk: int) -> HttpResponse:
+    inquiry = get_object_or_404(UnitInquiry, pk=pk)
+    if inquiry.student.user != request.user and not request.user.is_staff:
+        raise PermissionDenied("You are not authorized to cancel this inquiry.")
+    inquiry.status = "INQ_CANC"
```

Comment:
you need to modify the inquiry model in order for this to actually work, `status` is not just an arbitrary string